internal/status/status.go

```diff
@@ -15,17 +19,65 @@ func NewCommand() *cobra.Command {
 		Aliases: []string{"s"},
 		Short:   "show the current song or episode",
 		RunE: func(cmd *cobra.Command, _ []string) error {
+
+			watch, _ := cmd.Flags().GetBool("watch")
+
 			api, err := internal.Authenticate()
 			if err != nil {
 				return err
 			}
 
-			status, err := status(api)
-			if err != nil {
-				return err
+			if watch {
+
+				ticker := time.NewTicker(1000 * time.Millisecond)
+				stopWatching := make(chan bool)
+
+				go func() {
+					os.Stdin.Read([]byte{0x0})
+					stopWatching <- true
+				}()
+
+				go func() {
+					for {
+						select {
+						case <-stopWatching:
+							return
+						case <-ticker.C:
+							status, err := status(api)
+
+							if err != nil {
+								fmt.Println(err)
+								return
+							}
+
+							if runtime.GOOS == "windows" {
+								cmd := exec.Command("cmd", "/c", "cls")
```

Comment:
This line is tabbed over ~8 times. This function definitely needs to be split into several sub-functions.